src/locale/locales/ga/messages.po

```diff
@@ -20,192 +20,105 @@ msgstr "(tá ábhar leabaithe ann)"
 msgid "(no email)"
 msgstr "(gan ríomhphost)"
 
-#: src/view/com/notifications/FeedItem.tsx:232
-#: src/view/com/notifications/FeedItem.tsx:323
+#: src/view/com/notifications/FeedItem.tsx:294
 msgid "{0, plural, one {{formattedCount} other} other {{formattedCount} others}}"
 msgstr "{0, plural, one {duine amháin eile} two {beirt eile} few {{formattedCount} dhuine eile} many {{formattedCount} nduine eile} other {{formattedCount} duine eile}}"
 
-#: src/lib/hooks/useTimeAgo.ts:156
-msgid "{0, plural, one {# day} other {# days}}"
-msgstr ""
-
-#: src/lib/hooks/useTimeAgo.ts:146
-msgid "{0, plural, one {# hour} other {# hours}}"
-msgstr ""
```

Comment:
Some existing strings seem to be being removed in this PR, like here?

Reply:
Good catch, thanks!  Seems that I was working from an outdated POT file.  If it's OK, I'll just add the missing translations on this branch and ask to be merged then? Sorry for the confusion.

Reply:
No worries! Yeah there's a lot of churn in strings, they're being added and removed all the time, it's quite a job for us translators to keep up 😅

> If it's OK, I'll just add the missing translations on this branch and ask to be merged then?

I should note that I'm by no means an expert on this stuff, but I'm wondering if that would work given the changes in the code references?

I don't know if this would be possible/easy, but would adding the new translations to the current `messages.po` file be an option? That was generated yesterday in preparation for Monday's 1.91.1 release:

https://github.com/bluesky-social/social-app/blob/main/src/locale/locales/ga/messages.po

Btw, in case it's helpful, you can check how complete the translation will be for any language if a specific PR is merged:

- Look in the **All checks have passed** section at the bottom of any PR
- Click on the **Details** link for the **Lint / Run linters (pull_request)** entry
- Click the **Check & compile i18n line**

You'll then see a table which shows what the total number of translated strings for each language and the number of missing translations will be when the message catalogs are compiled as a result of the PR.

For this PR, for instance, [this is the link](https://github.com/bluesky-social/social-app/actions/runs/10863516241/job/30147745575?pr=5345), and clicking the **Check & compile i18n line** will show you this table:

![IMG_1982](https://github.com/user-attachments/assets/cb0cb5bc-dfd5-4b74-a3ea-c675322834be)

Reply:
Wow, this will be a huge help going forward, thanks.  And to be clearer than my post above, I'm actually resyncing my translation with the latest messages.po and not just adding the missing strings — the next commit (hopefully tonight) should have the correct references and so on.

Reply:
Excellent 👌

Glad I could help :)

Reply:
Oh and I don't think there's a huge rush btw, Paul usually merges translations right before he cuts a release, so it could be anywhere from a few days to a week plus before he does translation merges again.